src/tools/genpolicy/update_policy_samples.py

```diff
@@ -58,6 +58,14 @@ def timeRunCmd(arg):
 print("========== COMMAND: " + next_command)
 runCmd(next_command)
 
+print("Modifying settings for testing")
+runCmd("cp genpolicy-settings.json default-genpolicy-settings.json")
+runCmd("""jq '.request_defaults.CreateContainerRequest.allow_env_regex_map = {  
+  "JOB_COMPLETION_INDEX": "^[0-9]+$",
+  "CPU_LIMIT": "^[0-9]+$",
+  "MEMORY_LIMIT": "^[0-9]+$"
+}' genpolicy-settings.json > tmp-genpolicy-settings.json && mv tmp-genpolicy-settings.json genpolicy-settings.json""")
```

Comment:
Let's add a comment to mention this needs to be in sync with policy_test_cases.py in MAP.